modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -1329,7 +1345,7 @@ private void queueWatchEvent() {
                     eventCache = new ArrayList<>();
                 }
 
-                eventCache.add(List.copyOf(updatedEntries));
+                eventCache.add(updatedEntries.copy());
```

Comment:
Since we now have a separate class, we can unify `copy` and `clear` into a single call (e.g. `transfer`). I believe that they are always used together

Reply:
Clear still must be available to be called without copy

Reply:
missed that, you are correct